tui/installer/screens.py

```diff
@@ -482,10 +485,14 @@ def confirm_erase_volume_groups(answers):
     return RIGHT_FORWARDS
 
 def use_extra_media(answers):
+    message = "You installed Would you like to install any Supplemental Packs?"
```

Comment:
This doesn't seem to read correctly - should the `You installed` be there?

Reply:
Right, changed my mind change in the middle of it and missed that.